SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -210,6 +210,7 @@ USpatialActorChannel::USpatialActorChannel(const FObjectInitializer& ObjectIniti
 	, EventTracer(nullptr)
 	, LastPositionSinceUpdate(FVector::ZeroVector)
 	, TimeWhenPositionLastUpdated(0.0)
+	, bIsAutonomousProxyOnAuthority(false)
```

Comment:
Could you also set this to false in `USpatialActorChannel::Init`? Actor channels are pooled, and the same channel could get reused for another actor instead of being destroyed and re-constructed, but `Init` would be called in between to reset the state.

Reply:
Done!

Reply:
If we're storing this on the actor channel, will it get reset correct in the instance an actor has it's dormancy toggled and the channel is closed?

Reply:
I'm not sure, is there a way for me to test that?

Reply:
Update: Tested using an actor that toggled dormancy on and off and confirmed it was reset in the server. Accounting for this in SetChannelActor.